File: UltimateTicTacToe/Models/DTOs/MoveDTO.cs
using Newtonsoft.Json;
using UltimateTicTacToe.JsonConverters.Board;
using UltimateTicTacToe.Models.Game;

namespace UltimateTicTacToe.Models.DTOs
{
    public class MoveDTO
    {
        public BoardGame game;
        public Move move;
        public bool gameOver;
    }
}

File: UltimateTicTacToe/ClientApp/src/app/models/DTOs/MoveDTO.ts
import { Move } from "../move/move.model";
import { BoardGame } from "../boardGame/boardgame/boardgame.model";

export class MoveDTO {
    public lastMove: Move;
    public game: Array<Array<BoardGame>>;
    public UserId: number;
    public move: Move;
}
